deploy/tests/test.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'html/proofer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.